README.rst

```diff
@@ -91,7 +94,8 @@ Generators
 |                                          | sky130hd,          |                            |                                                                |                        
 |                                          | sky130hvl          |                            |                                                                |                                     
 +------------------------------------------+--------------------+----------------------------+----------------------------------------------------------------+
-| GDSFactory                               | sky130hd           |     No (In-progress)       | https://openfasoc.readthedocs.io/en/latest/flow-ldo.html       |
```

Comment:
Could add the README link under `Documentation`

Reply:
addressed in new commit